deeplake/core/transform/transform_dataset.py

```diff
@@ -1,40 +1,204 @@
-from deeplake.core.transform.transform_tensor import TransformTensor
+from deeplake.core.linked_tiled_sample import LinkedTiledSample
 from deeplake.util.exceptions import TensorDoesNotExistError
+from deeplake.core.partial_sample import PartialSample
+from deeplake.core.linked_sample import LinkedSample
+from deeplake.core.sample import Sample
+from deeplake.core.tensor import Tensor
+from typing import Union, List, Any
+from deeplake.constants import MB
+from itertools import chain
+
+import numpy as np
+
+import posixpath
+import bisect
+
+
+class TransformTensor:
+    def __init__(self, dataset, name, is_group=False):
```

Comment:
would prefer to have separate classes in separate files, our files tend to get really huge anyway